tilepix_test.go

```diff
@@ -82,3 +85,54 @@ func TestReadFile(t *testing.T) {
 		})
 	}
 }
+
+func TestRead(t *testing.T) {
+	tests := []struct {
+		name    string
+		input   io.Reader
+		dir     string
+		want    *tilepix.Map
+		wantErr bool
+	}{
+		{
+			name:    "Missing columns parameter in tileset",
+			input:   getInput(),
+			dir:     "testdata",
+			want:    nil,
+			wantErr: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			_, err := tilepix.Read(tt.input, tt.dir)
+
+			expectedError := "Tileset columns value not valid"
```

Comment:
Can you make this a field in the `tests` struct, so this test is more extensible